pkg/translator/azure/resourcelogs_to_logs.go

```diff
@@ -110,7 +110,6 @@ func (r ResourceLogsUnmarshaler) UnmarshalLogs(buf []byte) (plog.Logs, error) {
 			nanos, err := getTimestamp(log)
 			if err != nil {
 				r.Logger.Warn("Unable to convert timestamp from log", zap.String("timestamp", log.Time))
-				continue
```

Comment:
Looking at how this is handled elsewhere, I think we should not set the timestamp in this scenario and set the observed timestamp to the current time in both scenarios.

Reply:
> Looking at how this is handled elsewhere, I think we should not set the timestamp in this scenario and set the observed timestamp to the current time in both scenarios.

Good call. I'm posting the description of the field for documentation:

> Field: ObservedTimestamp
>
> Type: Timestamp, uint64 nanoseconds since Unix epoch.
> 
> Description: Time when the event was observed by the collection system. For events that originate in OpenTelemetry (e.g. using OpenTelemetry Logging SDK) this timestamp is typically set at the generation time and is equal to Timestamp. For events originating externally and collected by OpenTelemetry (e.g. using Collector) this is the time when OpenTelemetry’s code observed the event measured by the clock of the OpenTelemetry code. This field SHOULD be set once the event is observed by OpenTelemetry.
> 
> For converting OpenTelemetry log data to formats that support only one timestamp or when receiving OpenTelemetry log data by recipients that support only one timestamp internally the following logic is recommended:
> 
> Use Timestamp if it is present, otherwise use ObservedTimestamp.

Reply:
I think that looks correct based on how it's handled by the standard OTLP conversion and the Documentation. Nice work!